examples/montage/readme.md

```diff
@@ -1,49 +1,69 @@
+
+> 
+> _[MontageJS - http://montagestudio.com/montagejs/](http://montagestudio.com/montagejs/)_
 
 ## Learning MontageJS
-The [MontageJS](http://montagejs.org) website is a great resource for getting started.
+The [Montage Studio](http://montagestudio.com/montagejs/) website is a great resource for getting started.
 
 Here are some links you may find helpful:
 
-* [Quick Start](http://montagejs.org/docs/montagejs-setup.html)
-* [Demos](http://montagejs.org/docs/montagejs-examples.html)
-* [API Reference](http://montagejs.org/api/)
-* [Applications built with MontageJS](http://montagejs.org/apps/)
-* [Blog](http://montagejs.org/blog/)
-* [FAQ](http://montagejs.org/docs/faq.html)
+* [Quick Start](http://docs.montagestudio.com/montagejs/montagejs-setup.html)
+* [Demos](http://docs.montagestudio.com/montagejs/montagejs-examples.html)
+* [API Reference](http://docs.montagestudio.com/api/Component.html)
+* [Applications built with MontageJS](http://montagestudio.com/gallery/)
+* [Blog](http://montagestudio.com/blog/)
+* [FAQ](http://docs.montagestudio.com/montagejs/faq.html)
 * [MontageJS on GitHub](https://github.com/montagejs/montage)
 
 Articles and guides from the community:
 
-* [YouTube - Getting Started](http://www.youtube.com/watch?v=JfT1ML200JI)
+* [Screencast of SF HTML5 Live Code - Montage Studio with Ryan Paul](https://www.youtube.com/watch?v=aLT1ZmISKhw)
 * [My First MontageJS Application](http://renaun.com/blog/2013/05/my-first-montagejs-application/)
+* [Building a Simple Reddit Client with MontageJS](http://docs.montagestudio.com/montagejs/tutorial-reddit-client-with-montagejs.html)
+
 
 Get help from other Montage users:
 
-* [IRC](http://webchat.freenode.net/?channels=montage)
-* [MontageJS on Google Groups](https://groups.google.com/forum/?hl=en&fromgroups#!forum/montagejs)
-* [MontageJS on Twitter](http://twitter.com/montagejs)
-* [MontageJS on Google +](https://plus.google.com/116915300739108010954)
+* [Forum](http://forum.montagestudio.com/)
+* [Montage Studio on Twitter](http://twitter.com/montagejs)
+* [Montage Studio on Facebook](https://www.facebook.com/montagestudio)
+* [Montage Studio on Google +](https://plus.google.com/+Montagestudio/about)
```

Comment:
This reflects changes around the project in general, it's related to update the montage example in general, not just about the new css